### DISCUSSION TITLE
Version 2023-01-23

### DISCUSSION THREAD

**Discussion by DZrenner** [PredatH0r/ChanSort#345] [Q&A]
Ich vermisse in dieser Version den Befehl "Speichern unter..." incl. entsprechendem Button.

Bug oder Feature?

LG DZrenner

**Comment by PredatH0r** [comment]
Hi!

Diese Funktion habe ich absichtlich entfernt, da sie 
- bei vielen Listen (alle, die aus mehr als einer Datei bestehen) ohnehin gesperrt war
- von den meisten Fernsehern nur der Originaldateiname akzeptiert wird
- bei manchen Listen wichtige Informationen zum Dateiformat nur über den Dateinamen erkennbar sind
- ich oft Anfragen bekomme, wieso man bei "Speichern unter" keinen anderes Senderlistenformat auswählen kann

Jetzt muss man eben schon vorher im Windows-Explorer eine Kopie erstellen und diese dann bearbeiten.

**Reply by DZrenner** [reply]
Okay, danke für die Info.
Wenn man es weiß, kann man sich natürlich entsprechend arrangieren. "Speichern" und "Speichern unter" sind halt Standardbefehle, die man von (fast) allen Programmen her kennt und man verwendet die dann entsprechend. Ich habe damit sozusagen eine "Versionierung" und Archivierung mit Datum und dem Namen des Fernsehers vorgenommen.
Vielen Dank und viele Grüße
DZrenner

**Comment by devau** [comment]
Hallo DZrennerJa, diese Befehle vermisse ich auch! Es liegt hier doch am Benutzer zu wissen, was er danach macht mit den Dateien. Sehr schade, wurde das entfernt!DieterAm 03.02.2023 um 10:08 schrieb DZrenner ***@***.***>:
Okay, danke für die Info.
Wenn man es weiß, kann man sich natürlich entsprechend arrangieren. "Speichern" und "Speichern unter" sind halt Standardbefehle, die man von (fast) allen Programmen her kennt und man verwendet die dann entsprechend. Ich habe damit sozusagen eine "Versionierung" und Archivierung mit Datum und dem Namen des Fernsehers vorgenommen.
Vielen Dank und viele Grüße
DZrenner
